test/calibration/experiments/test_fine_frequency.py

```diff
@@ -0,0 +1,99 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test the fine frequency characterization and calibration experiments."""
+
+from test.base import QiskitExperimentsTestCase
+import numpy as np
+from ddt import ddt, data
+
+from qiskit.test.mock import FakeArmonk
+import qiskit.pulse as pulse
+
+from qiskit_experiments.library import (
+    FineFrequency,
+    FineFrequencyCal,
+)
+from qiskit_experiments.calibration_management.basis_gate_library import FixedFrequencyTransmon
+from qiskit_experiments.calibration_management import Calibrations
+from qiskit_experiments.test.mock_iq_backend import MockFineFreq
+
+
+@ddt
+class TestFineFreqEndToEnd(QiskitExperimentsTestCase):
+    """Test the fine freq experiment."""
+
+    def setUp(self):
+        """Setup for the test."""
+        super().setUp()
+        self.inst_map = pulse.InstructionScheduleMap()
+
+        self.sx_duration = 160
+
+        with pulse.build(name="sx") as sx_sxhed:
+            pulse.play(pulse.Gaussian(self.sx_duration, 0.5, 40), pulse.DriveChannel(0))
+
+        self.inst_map.add("sx", 0, sx_sxhed)
+
+        self.cals = Calibrations.from_backend(FakeArmonk(), FixedFrequencyTransmon())
+
+    @data(-0.5e6, -0.1e6, 0.1e6, 0.5e6)
+    def test_end_to_end_under_rotation(self, freq_shift):
```

Comment:
why only `under_rotation`?

Reply:
That should not be there. Fixed here: 96b88ce